apps/website/src/components/content/YouTubeCarousel.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import { Lightbox, Preview } from "@/components/content/YouTube";
+import Heading from "@/components/content/Heading";
+import { formatDateTime } from "@/utils/datetime";
+
+type Video = {
+  videoId: string;
+  title: string;
+  published: Date;
+};
+
+const YouTubeCarousel = ({ videos }: { videos: Video[] }) => {
```

Comment:
I wonder if you should accept a `dark` prop here like a few of our other components -- the font color used for the date is pretty unreadable when on the green background?